webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -38,6 +38,14 @@ def data_fits_data_model(data_model, stored_data):
     return True
 
 
+def data_fits_one_data_model(data_models, stored_data):
+    """
+    Method to find out whether the data entered by the user fits at least one model
+    """
+    fits_models = [data_fits_data_model(data_model, stored_data) for data_model in data_models]
+    return any(fits_models)
```

Comment:
You asked about a renaming, so maybe `data_fits_data_model_from/in_list` could be possible? But I think `data_fits_one_data_model` is also fine.